extensions/db-diagram/src/controllers/mainController.ts

```diff
@@ -29,14 +31,31 @@ export default class MainController extends ControllerBase {
 		return new Promise<boolean>(async (resolve) => {
 			vscode.commands.registerCommand('db-diagram.new', async () => {
 
-				// ...
-				console.log('new diagram');
-
-
+				//Dummy Data
+				/*const serverName = 'My Server';
+				const dbName = 'My Database';
+				const dbInformation = 'This is a database about a company. There is 1 significant cluster';
+				const tables = ['Employee', 'Department', 'Manager'];*/
+
+				// Create and show a new webview
+				const panel = vscode.window.createWebviewPanel(
+					'dbDiagram', // Identifies the type of the webview. Used internally
+					'DB Diagram', // Title of the panel displayed to the user
+					vscode.ViewColumn.One, // Editor column to show the new webview panel in.
+					{} // Webview options. More on these later.
+				);
+				const onDiskPath = vscode.Uri.file(
+					path.join('..', '..', 'src', 'ui-resources', 'index.html')
```

Comment:
This may not work when the extension has been webpacked, @Charles-Gagnon knows better about it though

Reply:
Not only that but src doesn't exist even in non-webpacked extensions (unless we're mistakenly including it)

You should use [ExtensionContext.ExtensionPath](https://github.com/microsoft/azuredatastudio/blob/main/src/vs/vscode.d.ts#L5556) as the base path and then put any non-TS stuff into a separate folder (ui-resources is fine) at the root level.